src/integrationTest/groovy/wooga/gradle/node/NodeReleasePluginIntegrationSpec.groovy

```diff
@@ -0,0 +1,45 @@
+package wooga.gradle.node
+
+import spock.lang.Unroll
+
+class NodeReleasePluginIntegrationSpec extends IntegrationSpec {
+
+    def setup() {
+        buildFile << """
+            ${applyPlugin(NodeReleasePlugin)}    
+        """.stripIndent()
+    }
+
+    @Unroll
+    def "converts #propertyName property value of #candidateName to #rcName"() {
+        given: "a gradle.properties file"
+        def gradleProperties = createFile("gradle.properties", projectDir)
+        gradleProperties << "${propertyName}=${candidateName}"
+
+        when:
+        def result = runTasksSuccessfully("properties")
+
+        then:
+        result.standardOutput.contains("${propertyName}: ${rcName}")
+
+        where:
+        propertyName = "release.stage"
+        candidateName = "candidate"
+        rcName = "rc"
+    }
+
+    @Unroll
+    def "renames task name from #candidateTaskName to #rcTaskName"() {
+        when:
+        // We expect the task to fail
+        def result = runTasks(candidateTaskName)
+
+        then:
+        !result.standardOutput.contains("task ':${candidateTaskName}'")
+        result.standardOutput.contains("task ':${rcTaskName}'")
```

Comment:
I would use:
```suggestion
        !result.wasExecuted(candidateTaskName)
        result.wasExecuted(rcTaskName)
```

Reply:
Yes! I forgot about this method.

Reply:
Though that fails since the run fails on something else, thus the right task is never executed.

```
> Task :modifyPackageJson_version FAILED
:modifyPackageJson_version (Thread[Execution worker for ':' Thread 3,5,main]) completed. Took 0.082 secs.
1 actionable task: 1 executed
 standardError=
FAILURE: Build failed with an exception.

Reply:
May have to revert to checking the list of tasks to be executed as before:

```
Tasks to be executed: [task ':modifyPackageJson_version', task ':nodeSetup', task ':npmSetup', task ':npm_publish', task ':node_publish', task ':githubPublish', task ':publish', task ':rc']

Reply:
then start it with a `--dry-run`?

Reply:
If I do so, they all get skipped like so:

```
Tasks to be executed: [task ':modifyPackageJson_version', task ':nodeSetup', task ':npmSetup', task ':npm_publish', task ':node_publish', task ':githubPublish', task ':publish', task ':rc']
Tasks that were excluded: []
:modifyPackageJson_version SKIPPED
:nodeSetup SKIPPED
:npmSetup SKIPPED
:npm_publish SKIPPED
:node_publish SKIPPED
:githubPublish SKIPPED
:publish SKIPPED
:rc SKIPPED

Reply:
yes but you can see that `rc` is in the list. Will the executed check still return true. I think so yes.

Reply:
I should have added that it did not when I tested it after you mentioned this:

```
Condition not satisfied:

result.wasExecuted(rcTaskName)
|      |           |
|      false       rc